src/styles/less/ParentNavTopBar.less

```diff
@@ -1,111 +1,108 @@
 .parent-nav-top-bar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 1% 7%;
+  margin-bottom: 2%;
+  border-bottom: 1px solid gray;
+  background-color: @turquoise-flat;
+  a {
+    color: black;
+  }
+  .navbar-logo {
+    margin: 0 0;
+    padding: 0 0;
+    font-family: 'Bangers';
+    font-style: normal;
+    font-weight: normal;
+    font-size: 42px;
```

Comment:
Can this also be a responsive unit like ems/rems to match other responsive items? Good use of percentages and responsive units throughout.